0002-telemetry-without-manual-instrumentation.md

```diff
@@ -0,0 +1,57 @@
+# (Open) Telemetry Without Manual Instrumentation
+
+_Cross-language requirements for automated approaches to extracting portable telemetry data with zero source code modification._
```

Comment:
can we also add injection of tracers?

Reply:
@AloisReitbauer can you be more specific about the requirement here? I can interpret "injection of tracers" in various ways and don't want to misunderstand/misrepresent.

Reply:
See comment below on specifying the trace at program startup and OpenTel being able to use this parameter to set - and load - the trace at runtime.